mitiq/interface/mitiq_qiskit/conversions.py

```diff
@@ -130,11 +130,14 @@ def _add_identity_to_idle(
     Returns:
         An unordered set of the indices that were altered
     """
+
     data = copy.deepcopy(circuit._data)
     bit_indices = set()
     idle_bit_indices = set()
     for op in data:
         gate, qubits, cbits = op
+        if gate.name == "barrier":  # Skip barriers
+            continue
```

Comment:
Is my understanding correct?

- Qiskit barriers have no Cirq analogue and so are removed in the Qiskit -> Cirq conversion.
- If a qubit in a Qiskit circuit only has a barrier acting on it, this will get removed in the Qiskit -> Cirq conversion, and so the qubit will be idle / not present in the Cirq circuit.

If so this LGTM. I wonder if there are more Qiskit operations which have no Cirq analogue and so would need to be treated in a similar manner here.

Reply:
Yes, your interpretation is correct. Basically, the scope of the `if gate.name == "barrier"` condition was to mark as idle qubit any qubit in which there are only "barrier" gates.

However I agree with you that it is more intuitive to simply remove barriers before calling this function. (See last commit).

Note that if there are other operations that have no Cirq analogue, the user will get a conversion error. Barriers represent a special case since they are intentionally removed by Mitiq without giving an error.